frontend/include/chpl/parsing/parsing-queries.h

```diff
@@ -434,8 +434,9 @@ void setAttributeToolNames(Context* context, AttributeToolNamesList keys);
 const AttributeToolNamesList& AttributeToolNames(Context* context);
 
 /**
-  Given an ID, returns the attributes associated with the ID. Only
-  declarations can have associated attributes.
+  Given an ID, returns the attributes associated with the ID. This is important
+  to use or else attributes for children of multiDecls or tupleDecls will not
+  be found.
  */
 const uast::AttributeGroup* idToAttributeGroup(Context* context, ID id);
```

Comment:
For symmetry and convenience, I think we should just go ahead and add a `attributeGroup()` that takes the AST.

Reply:
you mean something like `astToAttributeGroup(Context* context, const AstNode* node)`?